pandas-stubs/io/json/_json.pyi

```diff
@@ -82,7 +82,7 @@ class JSONTableWriter(FrameWriter):
 
 @overload
 def read_json(
-    path: FilePathOrBuffer,
+    path: FilePathOrBuffer = ...,
```

Comment:
This can't be right.  You have to specify at least one parameter to `read_json()`

Reply:
in 1.4 that is the case (but it certainly errors): https://github.com/pandas-dev/pandas/blob/015f67da78d4e8854d31bf628bb0af47b2e933a3/pandas/io/json/_json.py#L324

on main it has no default value anymore. I will remove the default value.